pom.xml

```diff
@@ -59,6 +59,8 @@
 		<el.version>3.0.0</el.version>
 		<box.version>2.30.1</box.version>
 		<jsoup.version>1.11.3</jsoup.version>
+		<chrome.driver.version></chrome.driver.version>
+		<firefox.driver.version></firefox.driver.version>
```

Comment:
I can delete it from here or leave it empty so it is visible that the property exists from the main pom.xml, what do you suggest @avano ?

Reply:
put a comment there that empty == latest

Reply:
I suggest adding it to the readme as well